sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DiagnosticsClientContext.java

```diff
@@ -77,7 +78,8 @@ public void serialize(DiagnosticsClientConfig clientConfig, JsonGenerator genera
                 }
                 generator.writeEndObject();
                 generator.writeStringField("consistencyCfg", clientConfig.consistencyRelatedConfig());
-                generator.writeStringField("proactiveInit", clientConfig.proactivelyInitializedContainersAsString);
+                generator.writeStringField("proactiveInitCfg", clientConfig.proactivelyInitializedContainersAsString);
```

Comment:
Some customers might already have taken dependency on this name, so we should be careful in future when coming up with these names. 
Not saying we can't do this, but would be great to follow a consistent naming pattern going forward.

Reply:
I think the key can end with`*Cfg` for naming consistency of client-level properties. I'll explicitly call out the above as a breaking change - while not ideal I believe it is okay to make this trade-off for 1 property and closely guard new property naming in future PRs.

Reply:
It is  not a breaking change - diagnostics are an opaque string - we should not make any compromises there.